res/controllers/Pioneer-DDJ-400.midi.xml

```diff
@@ -0,0 +1,4877 @@
+    </info>
+    <controller id="DDJ-400">
+        <scriptfiles>
+            <file filename="lodash.mixxx.js"/>
+            <file filename="midi-components-0.0.js"/>
+            <file functionprefix="PioneerDDJ400" filename="Pioneer-DDJ-400-script.js"/>
+        </scriptfiles>
+        <controls>
+            <!--
+                BROWSER Section START
+            -->
+            <control>
+                <description>BROWSE - rotate - Scroll tracklist/tree view</description>
+                <group>[Library]</group>
+                <key>MoveVertical</key>
+                <status>0xB6</status>
+                <midino>0x40</midino>
+                <options>
+                    <SelectKnob/>
+                </options>
+            </control>
+            <control>
+                <description>BROWSE +SHIFT - rotate - Zoom in/out of enlarged waveforms</description>
+                <group>[Master]</group>
+                <key>PioneerDDJ400.waveFormRotate</key>
+                <status>0xB6</status>
+                <midino>0x64</midino>
+                <options>
+                    <Script-Binding/>
+                </options>
+            </control>
+            <control>
+                <description>BROWSE - press - Move cursor between track list and tree view / open a folder</description>
+                <group>[Library]</group>
+                <key>MoveFocusForward</key> <!-- missing function to open Folder -->
+                <status>0x96</status>
+                <midino>0x41</midino>
+                <options>
+                    <Normal/>
+                </options>
+            </control>
+            <control>
+                <description>BROWSE +SHIFT - press - Move cursor between track list and tree view / close a folder
+                </description>
+                <group>[Library]</group>
+                <key>MoveFocusBackward</key> <!-- missing function to close Folder -->
+                <status>0x96</status>
+                <midino>0x42</midino>
+                <options>
+                    <Normal/>
+                </options>
+            </control>
+
+            <control>
+                <description>LOAD (DECK1) - press - Load a Track into Deck 1</description>
+                <group>[Channel1]</group>
+                <key>LoadSelectedTrack</key>
+                <status>0x96</status>
+                <midino>0x46</midino>
+                <options>
+                    <Normal/>
+                </options>
+            </control>
+            <control>
+                <!--
+                In Rekordbox LOAD+Shift (DECK1) function is related tracks, since we have no equivalent function in MIXXX
+                and doubleclick would map to instant double we are mapping this to CloneFromDeck
+                -->
+                <description>LOAD +SHIFT (DECK1) - press - Clone Track from playing Deck</description>
```

Comment:
According to the wiki page this should be "Sort library by criteria", not "Clone from deck".

Reply:
These parts are commented out anyway; we'll have to wait for 2.3.0 before reactivating.